### PR TITLE
completed javascript typing tutor

### PR DESCRIPTION
**Pull request by KiraHike**
image

### CODE REVIEW

File: javascript-typing-tutor/main.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+var $char = document.querySelectorAll('span');
+var $modal = document.querySelector('.modal');
+var $stats = document.querySelector('.stats');
+var $buttonYes = document.querySelector('.button-yes');
+var $buttonNo = document.querySelector('.button-no');
+var i = 0;
+var correct = 0;
+var incorrect = 0;
+var totalType = 0;
+var accuracy = 0;
```
Comment (thebearingedge) [inline javascript-typing-tutor/main.js:10]: Instead of declaring these here, just declare them right where you need them (`totalType`, `accuracy`).

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: javascript-typing-tutor/styles.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+span.gr {
+  color: green;
+}
+
+span.rd.ul {
+  color: red;
+  border-bottom: 5px solid red;
+}
```
[the remark this answers is not in the document]
Comment (KiraHike) [reply javascript-typing-tutor/styles.css:30]: sounds good

File: javascript-typing-tutor/main.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+function type(event) {
+  if ($char[i].textContent !== event.key) {
+    $char[i].className = 'span rd ul';
+    incorrect++;
+  } else if (i + 1 < $char.length) {
+    $char[i].className = 'span gr';
+    $char[i + 1].className = 'span bl ul';
```
[the remark this answers is not in the document]
Comment (KiraHike) [reply javascript-typing-tutor/main.js:18]: LOL...oops

File: javascript-typing-tutor/main.js (hunk 1 above)
Comment (KiraHike) [reply javascript-typing-tutor/main.js:10]: Okay :)

**Comment by KiraHike** [comment]
Changes made!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**